staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics/metrics.go

```diff
@@ -147,6 +147,17 @@ var (
 		[]string{"resource"},
 	)
 
+	WatchCacheReady = compbasemetrics.NewGaugeVec(
+		&compbasemetrics.GaugeOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "ready",
+			Help:           "Readiness of the watch cache broken by resource type.",
+			StabilityLevel: compbasemetrics.ALPHA,
+		},
+		[]string{"resource"},
```

Comment:
is this label bounded?

Reply:
it is theoretically unbounded but the number of values it takes is limited by the number of watch caches the apiserver creates. Because of that it is safe to add this label

Reply:
Most metrics in this file have this label, so I added a comment at the top.